kbase-extension/static/kbase/gallery/README.md

```diff
@@ -0,0 +1,44 @@
+# Gallery
```

Comment:
I'm dubious about adding this gallery functionality, especially since I can see it getting neglected and/or not used by anyone but you. I don't think I would bother to add stuff to it. @briehl your thoughts?

Reply:
I realize this, but honestly I added this because I don't know how anyone could review the changes without actually using them live in a browser, and this is the most direct way I could think of. I've had this code around for over a year, and every time I return to it I realize how useful it is. It has the benefits of storybook in that you can create any number of variations of a widget. (I think the last time I tried to get it in, that was BIll's comment - that storybook would eventually be added and take on the same responsibility). But this doesn't add any new dependencies, it is just a good ol static web app.